superset-frontend/package.json

```diff
@@ -124,8 +124,10 @@
     "d3-scale": "^2.1.2",
     "dom-to-image": "^2.6.0",
     "emotion-rgba": "0.0.9",
+    "fast-glob": "^3.2.7",
     "fontsource-fira-code": "^3.0.5",
     "fontsource-inter": "^3.0.5",
+    "fs-extra": "^10.0.0",
```

Comment:
What are these for? They seem to be newer versions than what's used in plugins, so I'm not sure of the relevance to this PR.

Reply:
used in `superset-frontend/scripts/copyAsserts.js`

Reply:
Ahh, I see. Should those be `devDependencies`, then?

Reply:
`eslint` suggest staying in dependecies,